openmetadata-ui/src/main/resources/ui/src/components/common/DBTConfigFormBuilder/DBTUpdateDescriptionField.component.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Space, Switch } from 'antd';
+import React from 'react';
+import { Field } from '../../Field/Field';
+
+interface Props {
+  dbtUpdateDescriptions: boolean;
+  id: string;
+  handleUpdateDescriptions: (value: boolean) => void;
+}
+
+function DBTUpdateDescriptionField({
```

Comment:
I would name this as `SwitchField` instead `DBTUpdateDescriptionField` since there are nothing specific to DBT we are doing. @aniketkatkar97 please rename this component

Reply:
Done